### PR TITLE
feat: add open by header in SfAccordion

### LINKED ISSUE #954: [FEATURE] Change firstOpen to open in SfAccordion
**Describe how this feature will help us improve**
Now you can open just firs accordion element on mount. Sometimes you want need to open some other tab.  

**Describe the solution you'd considered**
- change firstOpen to open
- add the possibility to set the title of the open element
- add the possibility to open a few elements on mount

### PR DESCRIPTION
**Pull request by martaradziszewska**
# Related issue

Closes #954 

# Scope of work

- add `open` prop 
- add the possibility to open more that one item by passing header or headers in array
- updated stories
- deprecated `firstOpen` to be removed in 1.0.0 due to the fact this change is breaking change

# Screenshots of visual changes

# Checklist

- [ ] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed

### CODE REVIEW

File: packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue [hunk 1]
```diff
@@ -14,8 +14,16 @@ Vue.component("SfAccordionItem", SfAccordionItem);
...
+    /**
+     * Opens an accordion item based on title
+     */
+    open: {
+      type: [String, Array],
+      default: ""
+    },
     /**
      * Opens the first accordion item if set to "true"
+     * @deprecated will be removed in 1.0.0 use open prop instead
```
Comment (pspaczek) [inline packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue:26]: @martaradziszewska can you create [RFC] for section how to deprecated element in our documentation?

File: packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue [hunk 2]
```diff
@@ -49,12 +57,24 @@ export default {
   },
   methods: {
     setAsOpen() {
-      if (this.$children.length) {
-        this.$children[0].isOpen = this.firstOpen;
+      if (this.$children && this.$children.length) {
+        // TODO remove in 1.0.0
```
Comment (pspaczek) [inline packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue:61]: maybe we can add begin and end?

Comment (martaradziszewska) [reply packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue:61]: sure, it's a good idea :) I'll add it

File: packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue (hunk 1 above)
Comment (martaradziszewska) [reply packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue:26]: yes I'll do it :)

**Comment by pspaczek** [comment]
@martaradziszewska 🎖

**Pull request merged**